packages/expo-document-picker/src/ExpoDocumentPicker.web.ts

```diff
@@ -21,7 +20,6 @@ export default {
     input.style.display = 'none';
     input.setAttribute('type', 'file');
     input.setAttribute('accept', Array.isArray(type) ? type.join(',') : type);
-    input.setAttribute('id', uuidv4());
```

Comment:
iirc I couldn't get this working on all platforms without setting the id attribute. Perhaps `Math.random()` or `Date.now()` would work instead?

Reply:
Do you have any idea on which platforms? I cannot se how having an `id` would affect, and I would be happy to investigate more 🤔 

Otherwise `String(Math.random())` should work!

Reply:
Updated to use `String(Math.random())` which produces strings with more than enough entropy for this use case:

<img width="175" alt="Screenshot 2022-12-15 at 14 52 06" src="https://user-images.githubusercontent.com/189580/207876025-bf621d34-4a6e-44cc-97a9-434f4f340bac.png">